rskj-core/src/main/java/co/rsk/rpc/modules/eth/subscribe/BlockHeader.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2017 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package co.rsk.rpc.modules.eth.subscribe;
+
+import org.ethereum.core.Block;
+import org.ethereum.rpc.TypeConverter;
+
+/**
+ * The block header DTO for JSON serialization purposes.
+ */
+public class BlockHeader {
```

Comment:
I would think on renaming this, mostly to avoid consuming mind cycles when you search for the `BlockHeader` and most of the time you would mean `org.ethereum.core.BlockHeader`. What about `BlockHeaderSubscription` or `BlockHeaderEvent`?